### ISSUE TITLE
Add Attribution to the Print Out

### ISSUE THREAD

**Issue by theduckylittle** [geomoose/gm3#152]
Specific data sets, suck as the OSM tiles, are meant to come with attribution.  This should make its way into the printout.

**Comment by klassenjs** [comment]
Possibly configure with a `<attribution>` element in the mapbook that could be used by both the interactive map and the PDF printout.

**Comment by tchaddad** [comment]
Looks like this issue is not in 3.2.2 beta - any chance of revisiting?

**Comment by theduckylittle** [comment]
I don't think this will make it into 3.2.2.  We've got a lot of fixes (and a few more enhancements) that should really go out ASAP.  Like, I feel we should stamp a release as soon as the current PRs are merged and tested.

I am going to keep this one as a "next up" priority as there is also a bug related to printing Cyrillic characters.  But I'd like to batch the print fixes as looking at that code, it could really use a few different reorginzations and clean-ups as well.
